core/src/main/java/bio/terra/pearl/core/model/ParticipantDashboardAlert.java

```diff
@@ -0,0 +1,22 @@
+package bio.terra.pearl.core.model;
+
+import java.util.UUID;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+//TODO: should these be versioned as well?
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+public class ParticipantDashboardAlert extends BaseEntity {
```

Comment:
I suspect we'll eventually have a parent "DashboardConfig" entity, and these will be attached to that, rather than the StudyEnvironment directly, but that's a task for another day